src/pages/LenderProfile/LenderProfile.vue

```diff
@@ -21,6 +21,12 @@
 			<lender-stats
 				:lender-info="lenderInfo"
 			/>
+
+			<lender-badges
```

Comment:
Looks like badges is between Loans and Dedications: https://github.com/kiva/kiva/blob/2fbbc6e05c4d0649e618bb499c4bb5c6789216bc/httpdocs/views/Lender/LenderView.tpl#L66